mw-webapp/src/component/title/Title.tsx

```diff
@@ -15,6 +15,9 @@ export enum HeadingLevel {
   h6 = "h6",
 }
 
+/**
+ * Title props
+ */
```

Comment:
maybe it would be good to add Title description @param @returns as in other components

Reply:
Added param but not returns because we don't write @returns for JSX.Element